examples/python/cloud-function-user-provision/main.py

```diff
@@ -0,0 +1,62 @@
+import requests
+import looker_sdk
+import os
+
+"""
+This Google Cloud Function takes in an email address and create
+a looker user based on another value provided. It could be easily adapted to accept an array
+of email addresses, to catch a webhook from a third party service, and much more.
+
+It could also easily be adapted to accept a second parameter and disable users as well.
+
+Example body: {"email": "test@test.com"}
+"""
+
+# We init the SDK in the "global scope" so that we don't have to do it on every execution.
+# This saves time if the function is called multiple times within a short timeframe.
+
+sdk = looker_sdk.init40()
+
+def create_user(request):
```

Comment:
Context: Came back here after going through the function. I think the name can be made more accurate, perhaps "create_user_or_reset_existing"?